vtparse/src/lib.rs

```diff
@@ -679,6 +679,14 @@ impl VTParser {
             self.parse_byte(*b, actor);
         }
     }
+
+    /// Force DCS pass through for tmux cc
+    pub fn force_dcs_passthrough(&mut self, bytes: &[u8], actor: &mut dyn VTActor) {
+        for b in bytes {
```

Comment:
Ah, this bit is in vtparse.  Layering wise this makes things very difficult.  I'd suggest doing this rerouting in the termwiz crate where we have the higher level state instead of here

Reply:
Understood. I'll try to think of something else. By the way I have an experimental branch tmux-cc at https://github.com/skyline75489/wezterm/tree/dev/tmux-cc-prototype, which handles the output part, very roughly but works. Take a look if you want :) 

(bear with all my amateur Rust code)